app/models/cdr_export.rb

```diff
@@ -0,0 +1,52 @@
+# == Schema Information
+#
+# Table name: cdr_exports
+#
+#  id           :integer          not null, primary key
+#  status       :string           not null
+#  fields       :string           default([]), is an Array
+#  filters      :json             not null
+#  callback_url :string
+#  created_at   :datetime
+#  updated_at   :datetime
```

Comment:
external_id uuid can be used for callback_url integration
when export ready system notify with

POST callback_url with external_id in body or something like this

Reply:
could you give me some details? we were discussing with @dmitry-sinina that GET will be enough & passing ID of export record in params to notify the client

Reply:
@gigorok  3d party system has it's own ids , uuids etc,

Reply:
yes, thats why after making POST request from 3d party system to yeti admin API (cdr_exports endpoint) we received export ID which have to be stored on 3d party database. in this case we can match it when callback_url will be touched